MANIFEST.in

```diff
@@ -9,3 +10,5 @@ recursive-include sphinx_rtd_theme *.svg
 recursive-include sphinx_rtd_theme *.ttf
 recursive-include sphinx_rtd_theme *.woff
 recursive-include sphinx_rtd_theme *.woff2
+recursive-include sphinx_rtd_theme/locale *.pot *.po *.mo
```

Comment:
Are mo files supposed to be distributed on Pypi? I thought they were built when sphinx runs.